lib/server.js

```diff
@@ -31,10 +31,10 @@ const BrowserCollection = require('./browser_collection')
 const EmitterWrapper = require('./emitter_wrapper')
 const processWrapper = new EmitterWrapper(process)
 
-const karmaJsPath = path.join(__dirname, '/../static/karma.js')
-const contextJsPath = path.join(__dirname, '/../static/context.js')
-
 function bundleResource (inPath, outPath) {
+  if (fs.existsSync(outPath)) {
```

Comment:
Moved responsibility to check if `outPath` file already exists to bundleResource method.

Reply:
Please put this comment in the git commit description.

Reply:
You mean pull request description ?